lib/phoenix/controller/pipeline.ex

```diff
@@ -23,10 +23,25 @@ defmodule Phoenix.Controller.Pipeline do
                  &(&1 |> Map.put(:phoenix_controller, __MODULE__)
                       |> Map.put(:phoenix_action, action))
 
-        Phoenix.Endpoint.instrument conn, :phoenix_controller_call,
+        start = System.monotonic_time()
+        :telemetry.execute([:phoenix, :controller, :call, :start], %{}, %{
```

Comment:
It'd be helpful to have an explicit `start_time` (non-monotonic) in this measurement